kafka-plugins-0.10/src/main/java/io/cdap/plugin/alertpublisher/KafkaAlertPublisher.java

```diff
@@ -167,11 +170,13 @@ private void validate() {
       try {
         Topic.validate(topic);
       } catch (InvalidTopicException e) {
-        throw new IllegalArgumentException(String.format("Topic name %s is not a valid kafka topic. Please provide " +
-                                                           "valid kafka topic name. %s", topic, e.getMessage()));
+        collector.addFailure(String.format(
+                "Topic name %s is not a valid kafka topic. Please provide valid kafka topic name. %s", topic,
```

Comment:
indentation is still off in this PR when there is a new line. This should either line up with the arguments from the previous line or be indented 2 spaces if its a method call.

```
    collector.addFailure(String.format(
      "Topic name %s is not a valid kafka topic. Please provide valid kafka topic name. %s", topic,
      e.getMessage()), null)
      .withConfigProperty(TOPIC);
```